src/app/clusters/ota-requestor/OTARequestor.cpp

```diff
@@ -388,12 +389,11 @@ void OTARequestor::ApplyUpdate()
 void OTARequestor::NotifyUpdateApplied(uint32_t version)
 {
     // New version is executing so update where applicable
-    VerifyOrReturn(Basic::Attributes::SoftwareVersion::Set(kRootEndpointId, version) == EMBER_ZCL_STATUS_SUCCESS);
```

Comment:
The original goal of this PR was to remove this line. The remaining changes is a result of the agreement that ConfigurationManager should now be the source of truth. I modified all direct calls to `Basic::Attributes::*::Get` to using `ConfigurationMgr().Get*()`